cardinal/plugin_persona.go

```diff
@@ -49,7 +50,7 @@ func (p *personaPlugin) Register(world *World) error {
 func (p *personaPlugin) RegisterQueries(world *World) error {
 	err := RegisterQuery[query.PersonaSignerQueryRequest, query.PersonaSignerQueryResponse](world, "signer",
 		query.PersonaSignerQuery,
-		WithCustomQueryGroup[query.PersonaSignerQueryRequest, query.PersonaSignerQueryResponse]("persona"))
+		query2.WithCustomQueryGroup[query.PersonaSignerQueryRequest, query.PersonaSignerQueryResponse]("persona"))
```

Comment:
a descriptive name would be better here to differentiate it from query rather then query2, imo.